### ISSUE TITLE
[keybinding] Keybinding collision between the debug console and the plug-in frontend view

### ISSUE THREAD

**Issue by kittaakos** [theia-ide/theia#3103]
```
root WARN Could not register keybinding:
  {"command":"pluginsView:toggle","keybinding":"ctrlcmd+shift+y"}
Error: "ctrlcmd+shift+y" is in collision with something else [scope:0]
```

https://github.com/theia-ide/theia/blob/8ee8b7476d85eb41a540631d464586fcca18e31a/packages/debug/src/browser/console/debug-console-contribution.ts#L34

https://github.com/theia-ide/theia/blob/8ee8b7476d85eb41a540631d464586fcca18e31a/packages/plugin-ext/src/main/browser/plugin-frontend-view-contribution.ts#L38

**Comment by akosyakov** [comment]
Oh, sorry, I used a keybinding from VS Code for the debug console. For extensions, in VS Code is `ctrlcmd+shift+x` is used, we could use it as well.

**Comment by vince-fugnitto** [comment]
Sorry just saw this now.
Closed with #3695, the keybinding <kbd>ctrlcmd</kbd>+<kbd>shift</kbd>+<kbd>x</kbd> was already being used by the extension view so a new one had to be used.
